src/components/victory-pie.jsx

```diff
@@ -243,7 +221,20 @@ class VPie extends React.Component {
   }
 
   render() {
-    const style = this.style.base;
+    if (this.props.animate) {
+      // Do less work by having `VictoryAnimation` tween only values that
+      // make sense to tween. In the future, allow customization of animated
+      // prop whitelist/blacklist?
+      const animateData = _.omit(this.props, [
+        "animate", "standalone"
+      ]);
+      return (
+        <VictoryAnimation {...this.props.animate} data={animateData}>
+          {props => <VictoryPie {...this.props} {...props} animate={null}/>}
+        </VictoryAnimation>
+      );
+    }
+    const style = this.style.parent;
     if (this.props.standalone) {
```

Comment:
Looks good! But let's clean up the code duplication at the bottom of `render()` while we're at it (for reference you could check out how I did it in `VictoryAxis`).